### PR TITLE
Address updates

### PR DESCRIPTION
**Pull request by navFooh**

### CODE REVIEW

File: components/CreateConfirm/index.tsx [hunk 1]
```diff
@@ -29,29 +29,19 @@ export const CreateConfirm = () => {
 
   // Get environment variables
   const { NEXT_PUBLIC_EXPLORER_URL: explorerUrl } = process.env
-  const { NEXT_PUBLIC_LOTUS_NODE_JSONRPC: apiAddress } = process.env
```
Comment (navFooh) [inline components/CreateConfirm/index.tsx:32]: We're not using this here anymore

File: components/CreateConfirm/index.tsx [hunk 2]
```diff
@@ -29,29 +29,19 @@ export const CreateConfirm = () => {
...
 
   // Get the message by cid
-  const {
-    data: messageData,
-    error: messageError,
-    loading: messageLoading
-  } = useMessageQuery({
+  const { data: messageData, error: messageError } = useMessageQuery({
```
Comment (navFooh) [inline components/CreateConfirm/index.tsx:34]: The `messageLoading` boolean was only passed as a dependency to the `useEffect` below, but not used

File: components/CreateConfirm/index.tsx [hunk 3]
```diff
@@ -29,29 +29,19 @@ export const CreateConfirm = () => {
...
     variables: { cid },
     pollInterval: 5000,
     // stop polling if we have a safe address
     skip: !!Address
   })
 
-  const error = useMemo(() => {
-    if (!!messageError && messageError.message.includes('key not  found'))
-      return messageError
-  }, [messageError])
```
Comment (navFooh) [inline components/CreateConfirm/index.tsx:49]: It seems like a mistake to only handle an error from the message query when it includes `'key.not..found'` with a double space between `not` and `found`

File: components/CreateConfirm/index.tsx [hunk 4]
```diff
@@ -29,29 +29,19 @@ export const CreateConfirm = () => {
...
-
   // Decode the receipt after receiving the message
   useEffect(() => {
     const receipt = messageData?.message?.receipt
-    if (!error && !!receipt?.return) {
+    if (!!receipt?.return) {
```
Comment (navFooh) [inline components/CreateConfirm/index.tsx:44]: If we have the return, it should be fine?

File: components/CreateConfirm/index.tsx [hunk 5]
```diff
@@ -66,15 +56,7 @@ export const CreateConfirm = () => {
...
-  }, [
-    messageData?.message,
-    apiAddress,
-    cid,
-    setMsigActor,
-    messageData,
-    messageLoading,
-    error
-  ])
+  }, [messageData?.message?.receipt, setMsigActor])
```
Comment (navFooh) [inline components/CreateConfirm/index.tsx:59]: Most of the dependencies seem excessive

File: components/CreateConfirm/index.tsx (hunk 4 above)
Comment (Schwartz10) [reply components/CreateConfirm/index.tsx:44]: This will change in an upcoming PR

**Review by Schwartz10 — APPROVED** [review]

**Pull request merged**